src/iomei.cpp

```diff
@@ -1453,6 +1473,28 @@ void MeiOutput::WriteMultiRpt(pugi::xml_node currentNode, MultiRpt *multiRpt)
     multiRpt->WriteNumbered(currentNode);
 }
 
+void MeiOutput::WriteNc(pugi::xml_node currentNode, Nc *nc)
+{
+    assert(nc);
+
+    WriteDurationInterface(currentNode, nc);
+    WritePitchInterface(currentNode, nc);
+    WritePositionInterface(currentNode, nc);
+
+    WriteLayerElement(currentNode, nc);
+    nc->WriteColor(currentNode);;
+    nc->WriteIntervalMelodic(currentNode);;
+    
+}
+
+void MeiOutput::WriteNeume(pugi::xml_node currentNode, Neume *neume)
+{
+    assert(neume);
+    neume->GenerateChildMelodic();
```

Comment:
I am quite skeptical with this call here. Writing should not change the data, there is a flaw in the design here.